ERR_WRONG_TYPE: universo (line 4, column 9) has the wrong type.
